signed-commit-and-push/action.yml

```diff
@@ -0,0 +1,31 @@
+name: "Commit and push changes with signed commit"
+description: "This action commits and pushes changes to a specified file with a signed commit message. The caller needs to set the following environment variables: FILE_TO_COMMIT, COMMIT_MESSAGE, and REPO."
```

Comment:
Why don't you take the inputs and set them in the "env" for the inline script instead?